### PR TITLE
Add Bitcoin Cash Node

### PR DESCRIPTION
**Pull request by ubunteroz**
Example SDL for running Bitcoin Cash Node on Bitcoin Cash mainnet

### CODE REVIEW

File: bitcoincashnode/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    bcash-profile:
+      resources:
+        cpu:
+          units: 4.0
+        memory:
+          size: 16Gi
+        storage:
+          size: 128Gi
+  placement:
+    westcoast:
```
Comment (boz) [inline bitcoincashnode/deploy.yaml:23]: Please add the `signedBy` section that you see in other examples.

**Review by boz — CHANGES_REQUESTED** [review]
Thanks so much @ubunteroz - this is great.

One change requested, this is good to go otherwise.

File: bitcoincashnode/deploy.yaml (hunk 1 above)
Comment (ubunteroz) [reply bitcoincashnode/deploy.yaml]: OK

**Review by 88plug — APPROVED** [review]
LOoks g

**Review by andy108369 — APPROVED** [review]

**Pull request merged**